Source/UserManagement/Concepts/PhoneNumber.cs

```diff
@@ -0,0 +1,36 @@
+using doLittle.Concepts;
+
+namespace Concepts
+{
+    public class PhoneNumber : Value<PhoneNumber>
+    {
+        public string Value { get; private set; }
+
+        public PhoneNumber(string value)
+        {
+            Value = value;
+        }
+
+        
+        //TODO: This value should default to false after the MVP and when there is logic for phone number confirmation
+        public bool Confirmed { get; private set; } = true;
+
+        public override bool Equals(object obj)
```

Comment:
There's no need to provide overrides for Equals and GetHashCode.  This is what Value<T> does.  It applies Value equality semantics (comparing on type and value of properties).